backtrader/feeds/psqlfeed

```diff
@@ -0,0 +1,86 @@
+# !/usr/bin/env python
+# -*- coding: utf-8; py-indent-offset:4 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from backtrader import DataBase, date2num
+import datetime as dt
+import urlparse
+import psycopg2
+
+class PsqlDatabase(DataBase):
+    '''
+    Fetches data from PSQL, and wraps it into a Feed consumable by cerebro
+    takes url connection string in form of :
+        postgres://{user}:{pw}@{host}:{port [optional]}/{database}
+
+    must also pass a datestart and a table name conforming to a similar schema as following:
+            ticker        char(5),
+            date          date,
+            high          numeric(10,4),
+            low           numeric(10,4),
+            open          numeric(10,4),
+            close         numeric(10,4),
+            volume        integer,
+            unique (ticker, date)
+    '''
+
+    params = (
+        ('url', None),
+        ('dataname', None),
+        ('table', None),
+        ('startdate', None),
+        ('ticker', None),
+    )
+
+    def start(self):
+        try:
+            cursor = self._connect_db()
+        except Exception as e:
+            print('Failed to establish connection to PSQL: {}'.format(e))
+            return
+
+        query = ("""SELECT date, high, low, open, close, volume """
+                 """FROM {table} """
+                 """WHERE ticker = '{ticker}' """
+                 """AND date >= '{startdate}' """
```

Comment:
The usage of `startdate` is inconsistent with other data feeds.  It also means that the `todate` parameter of the parent class is ignored, resulting in excessing data to be potentially fetched.

Please see the similar comments for the influxdb data feed in this community forum post: 

https://community.backtrader.com/topic/1279/resampling-in-influxdb-data-feed/3

Reply:
fine points!
- changed name startdate -> fromdate
- made fromdate optional
- added optional todate parameter

Reply:
>     * changed name startdate -> fromdate

`fromdate` and `todate` are already existing parameters defined in `AbstractDataBase`.  No need to define it in this class.

Reply:
removed